af_ktls.c

```diff
@@ -42,6 +42,8 @@
 #define KTLS_NONCE_SIZE			8
 
 #define KTLS_DATA_PAGES			(KTLS_MAX_PAYLOAD_SIZE / PAGE_SIZE)
+// Maximum value for msg_iovlen
+#define KTLS_SGL_MAX_ENTRIES    1024
```

Comment:
Use already defined constants if there is no need to introduce a custom one.
http://lxr.free-electrons.com/source/include/uapi/linux/uio.h